blog/_posts/2021-05-19-latest-stable.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "New 'latest-stable' branch in git repository"
+layout: archive
+author: Javier Lopez Gomez
+---
+
+This post is a must-read for advanced users that build ROOT from source. Starting with 6.24, we provide a new branch named `latest-stable` that will be regularly updated after each release. If you want to know more, keep reading!
+
+We have listened to you! We had users in the forum that were facing some issues building ROOT's `master` from source. However, in some circumstances the `master` branch might be temporarily broken on some platforms/compilers.
+
+To avoid this, the instructions to [build from source]({{'/install/build_from_source' | relative_url}}) suggested to clone the latest tagged release (or patches branch). This turns out to be inconvenient, as the tag/branch names for the latest available version change after each release, e.g., as of this writing, the latest tagged release is `v6-24-00` that will soon be superseded by `v6-26-00`. As a consequence, a user that wants to build from source the latest release, had to check the tag/branch name before issuing the `git clone` command.
+
+Starting with the 6.24 release, we created the `latest-stable` branch, which is targeted at users that regularly build ROOT from source. Furthermore, we will automatically provide updates to the latest release to users that already checked out this branch.
+
+## Great! How do I use it?
+Before getting hands-on, keep in mind that building from source is for advanced users. The preferred method for regular users to install ROOT is via pre-compiled packages. More on that can be found in the [install]({{'/install' | relative_url}}) guide.
+
+That said, our aim was to make this really simple for those who are used to build ROOT from source. Therefore, if you are interested in getting a copy of the latest stable version, you can do so as follows:
+```bash
+$ git clone --branch latest-stable https://github.com/root-project/root.git root_src
+```
+
+Then, you can follow the rest of the instructions in [build from source]({{'/install/build_from_source' | relative_url}}) as usual.
+
+But we didn't stop there. This branch will be updated regularly after each release, which means that you can easily upgrade ROOT to the latest release by simply:
+```bash
+$ git pull
+$ cd <empty build dir>
+$ # build as usual
+```
+
+## Summary
+In summary, we hope that this change saves time (and avoids issues) for users that build ROOT from source. If you face any problem while using this new branch, please feel free to report it [here](https://github.com/root-project/root/issues).
```

Comment:
Just a suggestion to avoid repeating the header:
```suggestion
We hope that this change saves time (and avoids issues) for users that build ROOT from source. If you face any problem while using this new branch, please feel free to report it [here](https://github.com/root-project/root/issues).
```